tests/garage/torch/modules/test_multi_headed_mlp_module.py

```diff
@@ -78,15 +78,16 @@ def helper_make_inits(val):
     'input_dim, output_dim, hidden_sizes, output_w_init_vals', plain_settings)
 def test_multi_headed_mlp_module(input_dim, output_dim, hidden_sizes,
                                  output_w_init_vals):
-    module = MultiHeadedMLPModule(
-        n_heads=len(output_w_init_vals),
-        input_dim=input_dim,
-        output_dims=output_dim,
-        hidden_sizes=hidden_sizes,
-        hidden_nonlinearity=None,
-        hidden_w_init=nn.init.ones_,
-        output_nonlinearities=None,
-        output_w_inits=list(map(helper_make_inits, output_w_init_vals)))
+    module = MultiHeadedMLPModule(n_heads=len(output_w_init_vals),
```

Comment:
would this new version of the test fail if the old bug was still there? if not, please add a case which would have failed.

Reply:
So I put hidden and output non-linear function in the `test_multi_headed_mlp_module_with_different_output_dims` test case.